beaker/data_model/experiment_spec.py

```diff
@@ -438,10 +438,10 @@ class TaskSpec(BaseModel, frozen=False):
     @field_validator("synchronized_start_timeout", "timeout", mode="before")
     @classmethod
     def ensure_nanoseconds(cls, value: Any) -> int:
-        if isinstance(value, (int, float)):
-            return int(value)
-        else:
+        if isinstance(value, str):
             return parse_duration(value)
+        else:
+            return value
```

Comment:
This used to cast floats to int. Is that still needed?

Reply:
pydantic will handle that on its own, but I'll add a test to be sure: a4b3bf0

Reply:
sgtm!